cookbook/request-user-for-authentication/README.md

```diff
@@ -0,0 +1,74 @@
+# Requesting a user to authenticate
+
+When your app requires authentication, you should invite the user to login when he tries to access your app features.
+The SignIn app provides a signin form ready to be invoked from other Starcounter apps thanks to the blending approach.
```

Comment:
I would rewrite this sentence to:

To separate the concerns, your app should not directly deal with user authentication. Rather than that, it should have a view that blends in an authentication view from another app that deals with that.

Below instruction shows how to provide an authentication notice that blends in the authentication form from the [SignIn](https://github.com/StarcounterApps/SignIn) app.